src/components/ClassScheduleList/index.js

```diff
@@ -49,17 +50,15 @@ class ClassScheduleList {
           defaultFilter.add(filter);
         });
 
-        classScheduleInstance.filtersData = [
-          {
-            filterKeys: ["course-slug"],
-            originalFilterKeys: ["course-slug"],
-            hightlight: false,
-            elements: [],
-            values: defaultFilter,
-          },
-        ];
-
-        classScheduleInstance.applyFilters();
+        this.classScheduleInstance.filtersData.push({
+          filterKeys: ["course-slug"],
+          originalFilterKeys: ["course-slug"],
+          hightlight: false,
```

Comment:
whoops, just noticed this. small typo krub
```suggestion
          highlight: false,
```

Reply:
Fixed in [07881e0](https://github.com/criclabs-co/mind-stretcher-web/pull/9/commits/07881e073262edf1165eaec9016fa6c4a2433a81) 👍